### PR TITLE
Entities crud operations

### PR DESCRIPTION
**Pull request by subash-cs**

### CODE REVIEW

File: src/test/java/org/shikshalokam/backend/scp/TestScpEntityTypesAndEntitiesCRUDOperations.java [hunk 1]
```diff
@@ -251,6 +251,87 @@ public void testDeleteEntityTypesWithValidPayload() {
...
+        // Log the status code and response body
+        int statusCode = response.getStatusCode();
+        response.prettyPrint();
+
+        // Validate response code is 201
+        Assert.assertEquals(statusCode, 201, "Status code should be 201");
+
+        logger.info("Ended calling the EntityCreation API with valid payload.");
+    }
+    @Test(dependsOnMethods = "testCreateEntityTypeWithInValidPayload", description = "Verifies the functionality of creating new entity with valid payload.")
```
Comment (infa-rvarahag) [inline src/test/java/org/shikshalokam/backend/scp/TestScpEntityTypesAndEntitiesCRUDOperations.java]: No need of this dependency

File: src/test/java/org/shikshalokam/backend/scp/TestScpEntityTypesAndEntitiesCRUDOperations.java [hunk 2]
```diff
@@ -251,6 +251,87 @@ public void testDeleteEntityTypesWithValidPayload() {
...
+        int statusCode = response.getStatusCode();
+        response.prettyPrint();
+
+        // Validate response code is 201
+        // Validate response code is 400 or 422 for invalid request
+        Assert.assertTrue(statusCode == 400 || statusCode == 422, "Status code should be 400 or 422 for invalid payload");
+
+        logger.info("Ended calling the EntityCreation API with invalid payload.");
+    }
+    @Test(dependsOnMethods = "testCreateEntityTypeWithEmptyFields", description = "Verifies the functionality of creating new entity with valid payload.")
```
Comment (infa-rvarahag) [inline src/test/java/org/shikshalokam/backend/scp/TestScpEntityTypesAndEntitiesCRUDOperations.java]: No need of this dependency

**Review by infa-rvarahag — CHANGES_REQUESTED** [review]

File: src/test/java/org/shikshalokam/backend/scp/TestScpEntityTypesAndEntitiesCRUDOperations.java (hunk 1 above)
Comment (subash-cs) [reply src/test/java/org/shikshalokam/backend/scp/TestScpEntityTypesAndEntitiesCRUDOperations.java]: done

File: src/test/java/org/shikshalokam/backend/scp/TestScpEntityTypesAndEntitiesCRUDOperations.java (hunk 2 above)
Comment (subash-cs) [reply src/test/java/org/shikshalokam/backend/scp/TestScpEntityTypesAndEntitiesCRUDOperations.java]: done

**Review by infa-rvarahag — APPROVED** [review]
Assert.assertTrue(statusCode == 400 || statusCode == 422, "Status code should be 400 or 422 for invalid payload");
  This type of validations not correct. Next submit correct the same .

**Comment by subash-cs** [comment]
> Assert.assertTrue(statusCode == 400 || statusCode == 422, "Status code should be 400 or 422 for invalid payload"); This type of validations not correct. Next submit correct the same .

Yeah,noted

**Pull request merged**